docs/README.md

```diff
@@ -117,6 +117,95 @@ Installation process
       helm install csi-baremetal csi/csi-baremetal-deployment --set driver.drivemgr.type=halmgr \
       --set global.registry=$REGISTRY --set global.registrySecret=$DOCKER_REGISTRY_SECRET
       ```
+Feature Supporting
+------
+### CIS hardening
+For supporting CIS Hardening we need to add the following settings while installing CSI and CSI Baremetal Operator:
+* While installing CSI Baremetal Operator set SecurityContext:
+  ```
+  --set securityContext.enable=true --set securityContext.runAsNonRoot=true --set securityContext.runAsUser=1000
+  ```
+* While installing CSI:
+  * Set the following SecurityContexts at deployments.csi-baremetal.dell.com for Controller, NodeController components:
+    ```
+    --set driver.controller.securityContext.enable=true --set driver.controller.securityContext.runAsNonRoot=true --set driver.controller.securityContext.runAsUser=1000 \
+    --set driver.securityContext.securityContext.enable=true --set driver.securityContext.securityContext.runAsNonRoot=true --set driver.securityContext.securityContext.runAsUser=1000
+    ```
+  * Create (if not existed) privileged PodSecurityPolicy:
+    ```
+    apiVersion: policy/v1beta1
+    kind: PodSecurityPolicy
+    metadata:
+      name: privileged
+      annotations:
+        seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
```

Comment:
Is this feature in alpha state?

Reply:
This feature is about to get out of alpha, but I didn't found this type of annotation in beta. Also, I just copied this definition from default psp privileged specification.